symfony/console/4.2/bin/console

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env php
+<?php
+
+use App\Kernel;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Component\Console\Input\ArgvInput;
+use Symfony\Component\Debug\Debug;
+use Symfony\Component\Dotenv\Dotenv;
+
+set_time_limit(0);
+
+require __DIR__.'/../vendor/autoload.php';
+
+if (!class_exists(Application::class)) {
+    throw new \RuntimeException('You need to add "symfony/framework-bundle" as a Composer dependency.');
+}
+
+if (class_exists(Dotenv::class)) {
+    (new Dotenv())->loadForEnv($_SERVER['APP_ENV'] ?? 'dev', __DIR__.'/../.env');
```

Comment:
what would be the outcome here if my `.env` had APP_ENV set to, say `STAGING` would `.env.dev` be loaded?

Reply:
No. .env and .env.STAGING will.